.travis.yml

```diff
@@ -28,6 +28,9 @@ before_install:
 install:
   - pip install $DJANGO
   - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install -U "djangorestframework<3.10"; fi
+  # Temporary: remove when openwisp-utils#71 is merged
+  - pip install "https://github.com/waleko/openwisp-utils/tarball/issues/40" --upgrade
```

Comment:
I released openwisp-utils 0.4.1 which contains your patch https://github.com/openwisp/openwisp-utils/pull/71.
You can now remove this temporary addition.